backend/fms_core/viewsets/study.py

```diff
@@ -55,4 +57,18 @@ def create(self, request, *args, **kwargs):
         else: 
             # Serialize study
             serializer = StudySerializer(study)
-            return Response(serializer.data)
\ No newline at end of file
+            return Response(serializer.data)
+
+    def destroy(self, request, pk=None):
+        errors = defaultdict(list)
+
+        if pk is None:
+            errors['Study'].append("pk cannot be None")
+        else:
+            _, errors['Removable'], _ = can_remove_study(pk)
+
+        if any(bool(error) for error in errors.values()):
+            raise ValidationError(errors)
+        else:
+            Study.objects.filter(pk=pk).delete()
```

Comment:
Check if it calls the Study overloaded delete() function. This could be calling the manager delete which will not create a version in reversion. If this is the case chain .first() before delete() to the model delete().